protorpc/messages.py

```diff
@@ -1134,19 +1134,19 @@ def insert(self, index, value):
     return list.insert(self, index, value)
 
 
-# TODO(rafek): Prevent additional field subclasses.
-class Field(object):
+class _FieldMeta(type):
 
-  __variant_to_type = {}
+  def __init__(cls, name, bases, dct):
+    getattr(cls, '_Field__variant_to_type').update(
+      (variant, cls) for variant in dct.get('VARIANTS', []))
+    type.__init__(cls, name, bases, dct)
 
-  class __metaclass__(type):
 
-    def __init__(cls, name, bases, dct):
-      getattr(cls, '_Field__variant_to_type').update(
-        (variant, cls) for variant in dct.get('VARIANTS', []))
-      type.__init__(cls, name, bases, dct)
+# TODO(rafek): Prevent additional field subclasses.
+class Field(six.with_metaclass(_FieldMeta, object)):
```

Comment:
we'd previously used `six.with_metaclass` elsewhere in this library, so i was just sticking with that.